header.php

```diff
@@ -52,11 +52,13 @@
 
   $logo_id = IGV_get_option('_igv_social_options', '_igv_metadata_logo_id');
```

Comment:
what microdata? the new header logo will be svg and loaded in with the theme. havent gotten to that yet.  the meta logo will be for footer and micro data whatever that is